src/python/doc/installation.rst

```diff
@@ -30,7 +30,8 @@ will be used by default, but you can force it by adding
 GUDHI Python module compilation
 ===============================
 
-To build the GUDHI Python module, run the following commands in a terminal:
+After making sure that the :ref:`Compilation dependencies` are properly installed,
```

Comment:
Just for curiosity, I think `` `Compilation dependencies`_`` would have worked without the new plugin.

Reply:
Yes I think so.

Reply:
I am just trying to judge the advantages of the plugin versus its drawbacks.

Reply:
Without the plugin, we have to do for instance:
```
:ref:`Perseus file format`
# and at the point of the documentation we want to reach
.. _Perseus file format:
```

We can also do:
```
`CGAL <installation.html#cgal>`__
```

With the plugin we have a behaviour much more like doxygen, where you have warnings on duplicate sections names. Tell me if you really dislike it I can remove it. It was maybe a bad choice.

Reply:
> Without the plugin, we have to do for instance:

Except when the reference is to a section in the same file, where section names are already automatic references without the plugin. That was the point of my comment.

> With the plugin we have a behaviour much more like doxygen, where you have warnings on duplicate sections names.

Oh, I didn't know we had that for doxygen. And it hasn't caused trouble yet? Hmm, maybe that's a sign that it isn't that bad...

> Tell me if you really dislike it I can remove it. It was maybe a bad choice.

I don't have a strong opinion. I am slightly worried, but if we have the same thing with doxygen, my worries may be unfounded. If you like the plugin, we can use it.

Reply:
The difference is that you can name section with doxygen, for instance:
```
\section mymoduleexample Example
```
You cannot do this with sphinx, you have to use anchors.